Same_func_twice.py

```diff
@@ -0,0 +1,20 @@
+def do_it_twice(func):
+    """
+    Make function executes twice.
+    :param func: The functopn to decorate
+    :return: decorator.
+    """
+    def inner(*args, **kwargs):
+        func(*args, **kwargs)
+        func(*args, **kwargs)
+    return inner
+
+
+@do_it_twice
+def add_numbers(x, y):
+    print("add number executed")
+    return x + y
+
+
+if __name__ == '__main__':
+    add_numbers(5, 3)
```

Comment:
This should not be here.
Don’t use git add -a ever, use git add -p
https://git-scm.com/docs/git-add